hivemq-edge/src/frontend/src/extensions/datahub/components/controls/DesignerToolbox.tsx

```diff
@@ -138,7 +142,15 @@ const DesignerToolbox: FC = () => {
                     <>
                       <Box pt={5} h="100%">
                         {activeStep === 0 && <ToolboxNodes />}
-                        {activeStep === 1 && <ToolboxDryRun onActiveStep={onActiveStep} />}
+                        {activeStep === 1 && (
+                          <ToolboxDryRun
+                            onActiveStep={onActiveStep}
+                            onShowNode={(node) => fitView({ nodes: [node], padding: 3, duration: 800 })}
```

Comment:
suggestion: Assuming that as _Edge_ grows, more time-based interactions and timeouts will keep appearing. For consistency, I suggest defining time constants that you'll be able to reuse.